### PR TITLE
Add a Multiselec for choosing Pupils to the "add class" form

### PR DESCRIPTION
**Pull request by Hrabarchuk**

### CODE REVIEW

File: src/YPS.Infrastructure/Services/MailSenderService.cs [hunk 1]
```diff
@@ -31,16 +45,31 @@ public async Task SendMessageAsync(string reciever,string subject,string text)
...
+                msg.IsBodyHtml = true;
+
+                var location = Assembly.GetEntryAssembly().Location;
+                var root = Path.GetDirectoryName(location);
+                var registration = File.ReadAllText(Path.Combine(root, _config.GetValue<string>(MailTemplateFolderKey), "Regitration.html"));
+
+                registration = registration.Replace("{{reciever}}", reciever)
+                    .Replace("{{password}}", password);
+
+
```
Comment (Redingston) [inline src/YPS.Infrastructure/Services/MailSenderService.cs]: Delete this

File: src/YPS.Infrastructure/Services/MailSenderService.cs [hunk 2]
```diff
@@ -31,16 +45,31 @@ public async Task SendMessageAsync(string reciever,string subject,string text)
...
+                msg.Body = registration;
+
```
Comment (Redingston) [inline src/YPS.Infrastructure/Services/MailSenderService.cs:63]: And this

File: src/YPS.Infrastructure/Services/MailSenderService.cs (hunk 1 above)
Comment (Redingston) [reply src/YPS.Infrastructure/Services/MailSenderService.cs]: row 63

File: src/YPS.Infrastructure/Services/MailSenderService.cs (hunk 2 above)
Comment (Redingston) [reply src/YPS.Infrastructure/Services/MailSenderService.cs:63]: row 65

**Review by Redingston — APPROVED** [review]

**Review by bodkia22 — APPROVED** [review]

**Review by vunt1k — APPROVED** [review]

**Pull request merged**